<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageTabGroup</class>
 <widget class="QWidget" name="PackageTabGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PackageTabGroup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_pTabW">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_pGeneralTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="PackageGeneral" name="m_pGeneralPage" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pContentTab">
      <attribute name="title">
       <string>Content</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PackageContent" name="m_pContentPage" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pAssociationsTab">
      <attribute name="title">
       <string>Associations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PackageAssoc" name="m_pAssociationsPage" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pDisplayTab">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PackageGeneral</class>
   <extends>QWidget</extends>
   <header>dock/pages/controller/packagegeneral.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PackageContent</class>
   <extends>QWidget</extends>
   <header>dock/pages/controller/packagecontent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PackageAssoc</class>
   <extends>QWidget</extends>
   <header>dock/pages/controller/packageassoc.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
